### DISCUSSION TITLE
맵끼리 연동되는 서비스는 언제쯤 나올 수 있을까요?

### DISCUSSION THREAD

**Discussion by hipsc159** [naverz/zepeto-studio-kor#661] [5. Q&A]
맵 용량 제한이 500mb라서 맵 여러개 만들고 연동시키고 싶은데 이러한 서비스가 언제 나올 수 있을지 궁금합니다!

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 구체적인 시기에 대한 확답은 어렵지만 지속적으로 니즈가 발생하는 이슈이기에 검토해 보도록 하겠습니다. 
감사합니다.

**Reply by hipsc159** [reply]
넵 감사합니다!
